scripts/docker_build.sh

```diff
@@ -17,8 +18,17 @@
 # SPDX-License-Identifier: Apache-2.0
 # License-Filename: LICENSE
 
+# Docker from ORT uses BuildKit feature
+DOCKER_BUILDKIT=1
+export DOCKER_BUILDKIT
+
 GIT_ROOT=$(git rev-parse --show-toplevel)
-GIT_REVISION=$(git describe --abbrev=10 --always --tags --dirty)
+GIT_VERSION=$(git describe --abbrev=7 --always --tags --dirty)
```

Comment:
BTW, the 7 needs to stay a 10 (to match what JitPack uses).

Reply:
Done on last commit